### ISSUE TITLE
Feature Request: execute command and quit

### ISSUE THREAD

**Issue by weaversa** [GaloisInc/cryptol#188]
The old Cryptol had a command line flag for executing Cryptol commands from the command line --- useful for hacking/scripting.

```
$cryptol-2]$ cryptol -h
Usage: cryptol [OPTION..] [fname]
...
 -c COMMAND  --command=COMMAND  execute command (or expression) and quit
...
```

Would it be possible to have this feature in the new version?

**Comment by acfoltzer** [comment]
This wouldn't be too hard to add for single commands. It might be nice to support multiple commands separated by semicolons or something, but that'll take a bit longer. Would those both be useful?

**Comment by weaversa** [comment]
The older version accepted multiple `-c` arguments on the command line, so, something along the lines of:

```
$ cryptol test.cry -c ":s sym" -c ":prove"
``

**Comment by weaversa** [comment]
Also, the location of the file denoted when it was loaded into Cryptol. This allowed commands to be issued to the interpreter before the file was loaded. So, the following will quiet the defaulting messages _before_ the file is loaded (key!)

```
$ cryptol -c ":s warnDefaulting=off" test.cry -c ":s prover=any" -c ":prove"
```

**Comment by acfoltzer** [comment]
Are there any use cases you have for command-before-load other than to adjust a `:set`ting? I'd rather just add flags for setting those and just have one way to feed commands. Making the order of command line arguments matter for this kind of thing is tricky at best.

**Comment by LeventErkok** [comment]
@weaversa I don't remember Cryptol-1 preserving the file order, but perhaps that was done after my retirement. That sounds fragile at best though, agreed with @acfoltzer Also, why not simply load from the command as well, if that's what you intend?

```
 $ cryptol -c ":s warnDefaulting=off" -c ":l test.cry" -c ":s prover=any" -c ":prove"
```

Note that keeping the order of arguments passed in "-c"'s is much easier than keeping the order of arguments and non-option arguments. (Because the former is essentially how getOpt works, so you get it for free.)

**Comment by weaversa** [comment]
Sounds good. Thanks!

**Comment by msaaltink** [comment]
I didn't even know this feature existed in cryptol-1!  It is almost as easy to write something like "echo -e :s warnDefaulting=off\\n:l test.cry\\n:s prover=any\\n:prove\\n:q | cryptol" (which I've done in some Makefiles), so I don't see a huge advantage to it.

**Comment by acfoltzer** [comment]
Implemented this in 31f0607264980afc420e61d4d55b15461795a8ca
